### ISSUE TITLE
incorrect 'git' version for GitPython==3.1.28 installed from PyPI

### ISSUE THREAD

**Issue by boegel** [gitpython-developers/GitPython#1500]
It looks like there's mistake in the release of GitPython 3.1.28 with respect to the version.

Here's the output I'm getting when GitPython 3.1.28 is installed (using `pip3 install`)

```
$ python3 -c 'import git; print(git.__version__)'
git
```

With older GitPython versions, I'm getting a proper version instead:

```
$ python3 -c 'import git; print(git.__version__)'
3.1.27
```

**Comment by branfosj** [comment]
In https://github.com/gitpython-developers/GitPython/commit/21ec529987d10e0010badd37f8da3274167d436f there is the change:
```diff
-__version__ = 'git'
+__version__ = "git"
```

However, [`setup.py`](https://github.com/gitpython-developers/GitPython/blob/main/setup.py#L50) puts the version in by doing a find/replace with a search for `'git'`:
```py
line = line.replace("'git'", "'%s'" % VERSION)
```
and that fails, which leaves `__version__ = "git"` in `__init__.py`.

**Comment by Byron** [comment]
Thanks for letting me know! This should be fixed with v3.1.29, please let me know in the comments if it isn't though.

**Comment by boegel** [comment]
I can confirm that a proper versio is printed for GitPython 3.1.29, thanks a lot for the quick fix!

```
$ pip3 install GitPython==3.1.29
$ python3 -c 'import git; print(git.__version__)'
3.1.29
```
